nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -152,7 +152,8 @@ language governing permissions and limitations under the License. -->
         </dependency>
         <dependency>
             <groupId>org.apache.activemq</groupId>
-            <artifactId>activemq-client</artifactId>
+            <artifactId>activemq-all</artifactId>
+            <scope>test</scope>
```

Comment:
No, we only need javax.jms at runtime. If we do need activeMq at runtime we are in big trouble unless the intention of JmsFactory and all components around it was to work with ActiveMq only.